hack/tool/backup_restore/internal/app/env.go

```diff
@@ -29,3 +36,18 @@ func GetAllENVs() map[string]string {
 	}
 	return envs
 }
+
+func GetTFBackendNSFromDeployment() string {
+	deployment := appsv1.Deployment{}
+	if err := K8SClient.Get(context.Background(), client.ObjectKey{Name: "terraform-controller", Namespace: "vela-system"}, &deployment); err != nil {
+		log.Printf("WARN: get terraform-controller deployment in the vela-system namesapce failed, %v", err)
+		return ""
+	}
+	envs := deployment.Spec.Template.Spec.Containers[0].Env
```

Comment:
Better if we check the container's name?

Reply:
Yes, you are right.